SS14.Launcher/ViewModels/ConnectingViewModel.cs

```diff
@@ -14,16 +14,19 @@ public class ConnectingViewModel : ViewModelBase
 
     private readonly CancellationTokenSource _cancelSource = new CancellationTokenSource();
 
+    private string? _overrideReason;
+
     public bool IsErrored => _connector.Status == Connector.ConnectionStatus.ConnectionFailed ||
                              _connector.Status == Connector.ConnectionStatus.UpdateError ||
                              _connector.Status == Connector.ConnectionStatus.ClientExited &&
                              _connector.ClientExitedBadly;
 
-    public ConnectingViewModel(Connector connector, MainWindowViewModel windowVm)
+    public ConnectingViewModel(Connector connector, MainWindowViewModel windowVm, string? overrideReason)
```

Comment:
I was thinking that instead of an override, this would be an *additional* line.

Reply:
Done.